Fitness/Views/Histogram.swift

```diff
@@ -167,31 +170,32 @@ class Histogram: UIView {
     func setupSelectedConstraints() {
 
         if selectedIndex < bars.count {
-            selectedLine.snp.remakeConstraints {make in
+            selectedLine.snp.remakeConstraints { make in
```

Comment:
remake or make?

Reply:
It's called anytime the user selects a bar, so it should be remake, right?